src/components/typography/typography.d.ts

```diff
@@ -50,7 +50,7 @@ export interface TypographyProps extends SpaceProps, ColorProps {
 
 declare function Typography(
   attrs: StyledComponentProps<
-    "div",
+    "span",
```

Comment:
while "span" looks correct to me based on the Typography component code, I'm not quite sure what's motivating this change as part of this PR on Card?